### ISSUE TITLE
Relocations table in MZ files not supported

### ISSUE THREAD

**Issue by VladRassokhin** [radare/radare2#1200]
Subj. Have old dos mz executable with relocation table.
Beginning of file:

```
[0x00000000 752 Downloads/motor-city/IDMAS.EXE]> x
- offset -  0 1  2 3  4 5  6 7  8 9  A B  C D  E F  0123456789ABCDEF
0000:0000  4d5a 7000 3b01 2709 4c02 774a 774a 412b  MZp.;.'.L.wJwJA+
0000:0010  0034 0000 4800 0000 1c00 0000 4b00 0000  .4..H.......K...
0000:0020  5000 0000 5500 0000 6600 0000 8400 0000  P...U...f.......
0000:0030  9900 0000 a300 0000 af00 0000 c600 0000  ................
0000:0040  de00 0000 e300 0000 f400 0000 f900 0000  ................
0000:0050  2401 0000 4201 0000 5601 0000 6801 0000  $...B...V...h...
```

As you can see relocation talbe starts at 0000:001c and contains 0x13b elements.

Radare 2 not processed relocation table, and after start comes into

```
[0000:0048]> x 10
- offset -  0 1  2 3  4 5  6 7  8 9  A B  C D  E F  0123456789ABCDEF
0000:0048  f400 0000 f900 0000 2401                 ........$.
```

which is part of relocation table.

**Comment by radare** [comment]
can we have  a test case for this?

**Comment by Maijin** [comment]
@marespiaut "can we have a test case for this?" ?

**Comment by marespiaut** [comment]
@Maijin It's not possible yet. First of all, because **Motor City** isn't a freeware game, so I have to find a freeware binary with the same problem. Second of all, I don't know (yet) what this is about, as I'm not familiar with the concept of _relocation table_.

**Comment by Maijin** [comment]
Well yeah no problem take something else ^^

**Comment by radare** [comment]
any other bin?

**Comment by marespiaut** [comment]
@radare Not for now. I'll do some research this evening.

**Comment by radare** [comment]
@marespiaut any update?

**Comment by VladRassokhin** [comment]
Good news, seems it was fixed in 0c57a435fc or bit later.

**Comment by Maijin** [comment]
@marespiaut ^
